app/src/main/java/org/bspb/smartbirds/pro/forms/convert/fields/SingleChoiceConverter.java

```diff
@@ -46,7 +41,7 @@ public void convert(Map<String, String> csv, JsonObject json, Set<String> usedCs
             json.add(jsonField, field);
         }
         usedCsvFields.add(csvField);
-        usedCsvFields.add(csvField + ".bg");
+        usedCsvFields.add(csvField + "." + locale);
```

Comment:
Would this produce warning/error if the language is changed between finalizing the monitoring and uploading?

Reply:
no